javascript/javascript3/week2/movies.js

```diff
@@ -0,0 +1,10 @@
+// Movies exercise : Bad Movies Array 
```

Comment:
I can see that the code is not formatted in this file.
I recommend you to use prettier extension for code formating at vscode
[prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode)
To know more about it, you can watch this video
[Code Formatting Extensions in VSCode](https://www.youtube.com/watch?v=iGWwmVmvFPk&ab_channel=SteveGriffith)

Reply:
i made the changes you recommended,Thank you so much